### ISSUE TITLE
[plugin-zod] Missing 2XX status keys of response objects in operations.ts file

### ISSUE THREAD

**Issue by assassinoNz** [kubb-labs/kubb#1428]
### What version of `kubb` is running?

3.0.11

### What kind of platform do you use?

Linux

### How does your `kubb.config.ts` config look like

```typescript
import { defineConfig } from "@kubb/core";
import { pluginOas } from "@kubb/plugin-oas";
import { pluginZod } from "@kubb/plugin-zod";

export default defineConfig({
    input: {
        path: "./typespec/tsp-output/@typespec/openapi3/openapi.json"
    },
    output: {
        path: "./kubb"
    },
    plugins: [
        pluginOas(),
        pluginZod({
            operations: true,
            coercion: {
                dates: true,
                numbers: true
            },
            unknownType: "unknown",
            dateType: "date",
            group: {
                type: "tag",
                name: ({ group }) => group
            },
            output: {
                path: "./zod",
                barrelType: false
            }
        })
    ]
});
```

### What version of external packages are you using(`@tanstack-query`, `MSW`, `React`, `Vue`, ...)

```JSON
"@kubb/cli": "^3.0.11",
"@kubb/core": "^3.0.11",
"@kubb/plugin-oas": "^3.0.11",
"@kubb/plugin-zod": "^3.0.11",
```

### What steps can reproduce the bug?

Use the provided kubb.config.ts with any OpenAPI schema which has at least one 2XX status response.

### How often does this bug happen?

Every time

### What is the expected behavior?

![image](https://github.com/user-attachments/assets/c7acb7bb-5a99-424a-a918-2f1d582ac554)

Here is a git diff of the operations.ts file generated by Kubb swagger-zod 2.28.4 (left) vs plugin-zod 3.0.11 (right).
Note how the latest version (right) drops the response with 2XX status code.

**Comment by stijnvanhulle** [comment]
@assassinoNz Fixed with version `3.0.12`

**Comment by assassinoNz** [comment]
@stijnvanhulle Thanks.

But you should probably remove this log as it pollutes the CLI output
https://github.com/kubb-labs/kubb/blob/44dfe05e468ad448324100be891db12cbf0becae/packages/plugin-oas/src/hooks/useOperationManager.ts#L152
